src/cli.rs

```diff
@@ -1130,6 +1136,22 @@ pub fn handle_cli_edge_app_command(command: &EdgeAppCommands) {
         EdgeAppCommands::GenerateMockData { path } => {
             let manifest_path = transform_edge_app_path_to_manifest(path);
             edge_app_command.generate_mock_data(&manifest_path).unwrap();
+        },
+        EdgeAppCommands::Validate { path } => {
+            let manifest_path = transform_edge_app_path_to_manifest(path);
+            match EdgeAppManifest::validate_file(&manifest_path) {
+                Ok(true) => {
+                    println!("Manifest file is valid.");
+                },
+                Ok(false) => {
+                    println!("Manifest file is invalid.");
```

Comment:
This is not good. More details... What is user supposed to do with this info?? You are not printing the actual error...

Reply:
Why is it even returning false? If it's not validated it should error. It should be called ensure_manifest_is_validated or something

Reply:
Printing error is inside the validate_file - I'll adjust it

Reply:
But why can your function still return an error then? Makes no sense to me tbh. If you already handle it inside your function